libcsm/logger.py

```diff
@@ -33,17 +31,16 @@
 
 
 class Logger(logging.Logger):
-    """
-    Inherits from logging.Logger, configuring custom settings on
-    initialization.
-    """
+    """Configure custom settings on initialization."""
```

Comment:
What's mandating the triple quotes on a single line, is that the `shortness`?

I _really_ don't like triple quotes on the same line.

Reply:
Probably due to triple quote strings being docstrings so the newlines would end up in autogenerated docs.

Reply:
I believe it comes from PEP https://peps.python.org/pep-0257/#one-line-docstrings. 

The strictness levels are:

- short (default): Expects a short summary and descriptions of arguments.
- long: Expects everything short expects, plus a longer description (if there is a newline in the docstring), and descriptions of return values and raised exceptions.
- full: Expects everything long expects, plus a description of yield values.

Reply:
`long` seems like it would like the new lines if I read that right..

Reply:
> Probably due to triple quote strings being docstrings so the newlines would end up in autogenerated docs.

For single-line things? I didn't notice anything weird in the sphinx docs https://github.com/Cray-HPE/libCSM/pull/37

Reply:
`long` did not fix it.  I can just ignore D200.  Looks like the latest merges need a lot of fixing up though still to conform to the new linter